src/Cli/dotnet/commands/dotnet-run/Program.cs

```diff
@@ -17,6 +17,14 @@ public static RunCommand FromArgs(string[] args)
             return FromParseResult(parseResult);
         }
 
+        internal static bool IsBinLogArgument(string arg)
+        {
+            const StringComparison comp = StringComparison.Ordinal;
+            return arg.StartsWith("/bl:", comp) || arg.Equals("/bl", comp)
+                || arg.StartsWith("--binaryLogger:", comp) || arg.Equals("--binaryLogger", comp)
+                || arg.StartsWith("-bl:", comp) || arg.Equals("-bl", comp);
+        }
```

Comment:
We should do what we should have done in the first place and made a proper S.CL `Option<string>` for the binlog argument with the grammar managed inside it like we've done for the `/property` option. I'd love to see that done (so we could reuse it).

Reply:
I don't know how to create `CliOption` that would parse `-bl` arguments properly. For example, `-bl:arg` should be valid whereas `-bl arg` should not (that should be parsed as two separate arguments). Similarly, it looks like `CliOption`s cannot be made case-insensitive.